nuxeo-distribution/nuxeo-functional-tests/src/main/java/org/nuxeo/functionaltests/RestHelper.java

```diff
@@ -571,4 +571,13 @@ protected static boolean executeHTTP(Supplier<Response> fetcher) {
         }
     }
 
+    /**
+     * @since 11.1
+     */
+    public static Map<String, Object> fetchDocumentProperties(String docId, List<String> schemas) {
+        String[] properties = schemas != null ? schemas.toArray(new String[0]) : new String[0];
+        Document document = CLIENT.repository().schemas("dublincore", properties).fetchDocumentById(docId);
```

Comment:
What do you think about putting `schemas("*")` on `CLIENT` initialization and not having to deal with schemas here?

Reply:
yes, i was just wondering about a perfs issue when we load the whole schemas. But i think in our hotreload test there is no a lot data loaded into Nuxeo.

Reply:
You're right about performance concern, but here for test I think it's quite normal to retrieve everything if we have the need.